timemachine/fe/free_energy.py

```diff
@@ -659,7 +676,14 @@ def compute_intermediate_result(lambdas: Sequence[float]) -> PairBarResult:
             print(f"Sampling new state at λ={lamb_new:.3g}…")
 
         lambdas = lambdas_new
-        results.append(compute_intermediate_result(lambdas))
+        result = compute_intermediate_result(lambdas)
+        results.append(result)
+    else:
+        warn(
```

Comment:
nit: Will this show a warning for min_overlap = 1?

Reply:
Yes. I was on the fence about making `min_overlap: Optional[float]`, in which case we could not warn if set to `None`. Maybe that is the preferable approach.

Reply:
Updated to make `min_overlap` optional, and not warn with the default setting of `None` in https://github.com/proteneer/timemachine/pull/1081/commits/d368e05afec6db35834abfd27a54c4ec694c8ce6